uiowa/uiowa
# Hotfixes for aggregator style options

Made assumptions in code and didn't account for existing blocks.

# To Test

pull, sandbox site, after update hook check http://sandbox.local.drupal.uiowa.edu/aggregator-0
besides broken images if you don't have them locally you should see more or less the same look and feel as https://sandbox.prod.drupal.uiowa.edu/aggregator-0
